### ISSUE TITLE
ExpansionPanel expanded max-height

### ISSUE THREAD

**Issue by esso23** [SamProf/MatBlazor#477]
In file matAccordion.scss there is this:
```cs
    &.mat-expansion-panel--expanded
    {
      > .mat-expansion-panel__content, .mat-expansion-panel__actions
      {
        max-height: 1000px;
        opacity: 1;
        display: block;
      }
    }
```

The max-height should be removed. It causes problems in NavMenu when there is a `MatNavSubMenuList` with many `MatNavItem`s you are unable to scroll to them. When I removed max-height everything was fine.

Edit: Ok, the solution I proposed is wrong. What works for me is setting max-height to 10000px (or any other high value). When I removed max-height, the expansion panel never displays.

**Comment by enkodellc** [comment]
@esso23 so are you suggesting to make it max-height: 10000px instead of max-height:1000px ?  
Because your screen is tall the 1000px is not a big enough value?

**Comment by lindespang** [comment]
Closing this one since PR was merged in release 2.4.3!
